dts/bindings/memory-controllers/st,stm32-fmc-nor-psram.yaml

```diff
@@ -111,11 +114,10 @@ child-binding:
         * EXTMOD - Extended mode enable.
                    If set, then 'st,timing-ext' shall be provided.
         * ASYNCWAIT -  Wait signal during asynchronous transfers.
-        * CPSIZE - Cellular RAM (CRAM) 1.5 Page Size.
         * CBURSTRW - Write burst enable.
         * CCLKEN - Continuous Clock Enable.
         * WFDIS - Write FIFO Disable.
-        * BMAP - FMC bank mapping.
+        * CPSIZE - Cellular RAM 1.5 Page Size.
```

Comment:
`BMAP - FMC bank mapping` and `FACCEN - Flash access enable` seem removed on purpose. I think it deserves a comment in the commit message, e.g.
```patch
-Description of parameters in st,control property didn't match the order
+Description of parameters in st,control property didn't match the values
 used in the code.
```